core/game_object.lua

```diff
@@ -22,14 +22,11 @@ function loadAPIs()
     function SMODS.GameObject:__call(o)
         o = o or {}
         o.mod = SMODS.current_mod
-        if o.mod and not o.raw_atlas_key and not (o.mod.omit_mod_prefix or o.omit_mod_prefix) then
+        if o.mod and not o.raw_atlas_key and not (o.mod.omit_mod_prefix or o.omit_mod_prefix) and not o['shader'] then
```

Comment:
Why should a `shader` being present prevent atlas keys from being prefixed?

Reply:
If the object is a shader, the atlas refers to how it appears in the collection only. Removing the mod prefix in that situation lets you use whichever atlas you like to display the shader, otherwise it’s assigned the mod prefix automatically